src/main/java/jp/ac/keio/bio/fun/xitosbml/CuiMainImgSpatial.java

```diff
@@ -0,0 +1,78 @@
+package jp.ac.keio.bio.fun.xitosbml;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.sbml.jsbml.SBMLException;
+import org.sbml.jsbml.SBMLWriter;
+import org.sbml.jsbml.TidySBMLWriter;
+
+import ij.IJ;
+import ij.ImagePlus;
+import jp.ac.keio.bio.fun.xitosbml.util.ModelValidator;
+import jp.ac.keio.bio.fun.xitosbml.xitosbml.SpatialSBMLExporter;
+
+public class CuiMainImgSpatial extends CuiMainSpatial {
+
+	protected ImagePlus imager;
+
+	public void run(String arg)
+	{
+		//empty
+	}
+	
+	public void runCui(String path1, String path2) {
```

Comment:
Rename `path1`, `path2` to a meaningful name

Reply:
Should I rename path1 and path2 to imagePath and outputPath respectively?

Reply:
Please refer to https://github.com/Medha-B/XitoSBML/issues/2.